browser/css/jsdialogs.css

```diff
@@ -42,8 +42,7 @@
 }
 
 img.ui-drawing-area {
-	width: -webkit-fill-available;
-	width: -moz-available;
```

Comment:
this breaks writer -> format -> theme (it is cut off)

Reply:
fill available makes things blurry...  yes, please remove it

I think we can fix theme dialog case by adding:
`width: max-content;` to `.jsdialog.ui-scrollwindow`

and that is all we need :)
you can remove #suggestionslb rule and this img.ui-frawing-area rule

Reply:
ahh right !